src/test/ipv-stub/ipv-user-identity-response.test.ts

```diff
@@ -0,0 +1,92 @@
+  ],
+  "https://vocab.account.gov.uk/v1/coreIdentity": {
+    name: [
+      {
+        nameParts: [
+          {
+            type: "GivenName",
+            value: "kenneth",
+          },
+          {
+            type: "FamilyName",
+            value: "decerqueira",
+          },
+        ],
+      },
+    ],
+    birthDate: [
+      {
+        value: "1964-11-07",
+      },
+    ],
+  },
+  "https://vocab.account.gov.uk/v1/address": [
+    {
+      addressCountry: "GB",
+      uprn: null,
+      buildingName: "",
+      streetName: "HADLEY ROAD",
+      postalCode: "BA2 5AA",
+      buildingNumber: "8",
+      addressLocality: "BATH",
+      validFrom: "2000-01-01",
+    },
+  ],
+  "https://vocab.account.gov.uk/v1/drivingPermit": [
+    {
+      personalNumber: "DOE99802085J99KV",
+      fullAddress: "122 BURNS CRESCENT EDINBURGH EH1 9GP",
+      issueNumber: "16",
+      issuedBy: "DVLA",
+      issueDate: "2019-01-23",
+      expiryDate: "2023-01-18",
+    },
+  ],
+  "https://vocab.account.gov.uk/v1/socialSecurityRecord": [
+    {
+      socialSecurityRecord: [
+        {
+          personalNumber: "LR850265D",
+        },
+      ],
+    },
+  ],
+  "https://vocab.account.gov.uk/v1/passport": [
+    {
+      documentNumber: "1223456",
+      expiryDate: "2022-02-02",
+    },
+  ],
+  "https://vocab.account.gov.uk/v1/returnCode": [
+    {
+      code: "A",
+    },
+    {
+      code: "P",
+    },
+  ],
+};
```

Comment:
This looks like the same object as the one in the lambda code. Could we move this example object into a test utils file or just a utils files  and have the test and code reference it.

Reply:
We could, but it didn't seem right to me to reference the hard coded return value from the test because if someone changed it and it became invalid, the test would still pass. The hard coded value will be removed in a later ticket but it may still be convenient to the json string in the test file.

Reply:
Done